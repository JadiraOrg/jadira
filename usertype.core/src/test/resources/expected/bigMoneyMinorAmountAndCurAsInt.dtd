<!ELEMENT dataset (
    BIGMONEYMINORAMOUNTANDCURASINT*)>

<!ELEMENT BIGMONEYMINORAMOUNTANDCURASINT EMPTY>
<!ATTLIST BIGMONEYMINORAMOUNTANDCURASINT
    ID CDATA #REQUIRED
    MY_CURRENCY CDATA #IMPLIED
    MY_AMOUNT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

