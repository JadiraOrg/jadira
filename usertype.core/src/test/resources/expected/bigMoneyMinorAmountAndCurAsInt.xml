<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "bigMoneyMinorAmountAndCurAsInt.dtd">
<dataset>
  <bigMoneyMinorAmountAndCurAsInt ID="0" MY_CURRENCY="840" MY_AMOUNT="10000" NAME="test_0"/>
  <bigMoneyMinorAmountAndCurAsInt ID="1" MY_CURRENCY="840" MY_AMOUNT="10010" NAME="test_1"/>
  <bigMoneyMinorAmountAndCurAsInt ID="2" MY_CURRENCY="978" MY_AMOUNT="99" NAME="test_2"/>
  <bigMoneyMinorAmountAndCurAsInt ID="3" MY_CURRENCY="978" MY_AMOUNT="-99" NAME="test_3"/>
  <bigMoneyMinorAmountAndCurAsInt ID="4" NAME="test_4"/>
</dataset>